kubetest/kind/kind.go

```diff
@@ -261,6 +261,7 @@ func (d *Deployer) Build() error {
 
 	// Build the node image (including kubernetes)
 	cmd := exec.Command("kind", args...)
+	cmd.Dir = d.importPathK8s
```

Comment:
imho we should specify `--kube-root` to build instead, which doesn't rely on how well kind can detect the kube root automatically.